schema.gql

```diff
@@ -21,8 +27,11 @@ input LoginInput {
 }
 
 type Mutation {
+  changePassword(data: ChangePasswordInput!): User!
```

Comment:
Even though we are using types, we should still use thoughtful variable names for arguments, so I would probably change `data` to `changePasswordInput`

Reply:
Updated changePasswordInput and resetPasswordInput